src/Microsoft.FeatureManagement/Targeting/Audience.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+//
+using System.Collections.Generic;
+
+namespace Microsoft.FeatureManagement.FeatureFilters
+{
+    /// <summary>
+    /// An audience definition describing a set of users.
+    /// </summary>
+    public class Audience
+    {
+        /// <summary>
+        /// Includes users in the audience by name.
+        /// </summary>
+        public List<string> Users { get; set; }
+
+        /// <summary>
+        /// Includes users in the audience based off a group rollout.
+        /// </summary>
+        public List<GroupRollout> Groups { get; set; }
+
+        /// <summary>
+        /// Includes users in the audience based off a percentage of the total possible audience.
```

Comment:
Can we be clear the expected value is 0..1 instead of 0..100?

Reply:
I'm confused here. The expected value is in the range [0,100]. It sounds like you're saying we should do [0,1].

Reply:
Since it's double now (instead of int), it can be confusing about the range. Yes, I think we should do [0,1]. The portal can do the necessary conversion.

Reply:
I would be confused if it was anything other than out of 100 since that's what percentage means. per one hundred.

Reply:
Ok, I don't feel too strongly about this. You do have a point. Can we just clarify the range in the comment?

Reply:
Absolutely, I should clarify it.